packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -207,11 +200,6 @@ export const prepareVaultManagerKit = (
   );
 
   const poolIncrementSeat = provideEmptySeat(zcf, baggage, 'pool increment');
-  const retainedCollateralSeat = provideEmptySeat(
```

Comment:
will the new liquidation system need something like this?

Reply:
Yes, it re-appears in #7074. This was a quick way to make lint happy.

Reply:
consider a lint suppression since it's temporary. that would retain the blame history